core/src/idx/ft/analyzer/mapper.rs

```diff
@@ -0,0 +1,21 @@
+use crate::err::Error;
+use crate::idx::ft::analyzer::filter::FilterResult;
+use std::path::Path;
+use std::sync::Arc;
+use vart::art::Tree;
+use vart::VariableSizeKey;
+
+#[derive(Clone, Default)]
+pub(in crate::idx) struct Mapper {
+	_terms: Arc<Tree<VariableSizeKey, String>>,
+}
+
+impl Mapper {
+	pub(in crate::idx) fn new(_path: &Path) -> Result<Self, Error> {
+		todo!()
+	}
+
+	pub(super) fn map(&self, _s: &str) -> FilterResult {
```

Comment:
Hey @emmanuel-keller 

There are functions available on vart which avoid the CoW approach, and would be faster for insertion. Just wanted to update. Improving the [docs](https://github.com/surrealdb/vart/pull/58) to make it better